packages/rocketchat-language-plugin/server/leave_automatic_channel.js

```diff
@@ -0,0 +1,18 @@
+Meteor.methods({
+	leave_automatic_channel(room_name) {
```

Comment:
disable channel allows admin to disable the leaving of a channel, but leave_automatic_channel blacklists a channel for that user.